cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -543,6 +543,45 @@ For Kerberos-enabled Hadoop clusters:
 
   Restart CDAP after removing the usercache(s).
 
+
+.. _cloudera-configuration-integration-sentry:
+
+Enabling Sentry
+---------------
+To use CDAP with Cloudera clusters using Sentry authorization, these properties need to be set:
+
+- Add the user ``cdap`` to the Hive property ``sentry.metastore.service.users``
+- Add the user ``cdap`` to the Sentry property ``sentry.service.admin.group``
+- Add the user ``cdap`` to the Sentry property ``sentry.service.allow.connect``
+
+The Hive properties are set in ``hive-site.xml``, while the Sentry properties are set in
+``sentry-site.xml``.
+
+These can be set from within Cloudera Manager by using the appropriate "safety valve":
+
+- For Hive, use the *Hive Service Advanced Configuration Snippet (Safety Valve) for
+  hive-site.xml* (API ``hive_service_config_safety_valve``), setting it as::
+
+    <property>
+        <name>sentry.metastore.service.users</name>
+        <value>hive,impala,hue,hdfs,cdap</value>
+    </property>
```

Comment:
This is incorrect. None of these are safety valves. The `sentry.metastore.service.users` setting is in Hive on clusters with Sentry installed.
<img width="815" alt="screen shot 2016-08-10 at 5 08 57 pm" src="https://cloud.githubusercontent.com/assets/380021/17571199/3934fe66-5f1d-11e6-9ae9-d03c9166eb95.png">

Also, this setting should *only* be set if the user wishes to allow CDAP to bypass Sentry authorization for Hive Metastore queries, such as in the default case where all applications run as `cdap` and not individual users.